source/features/view-markdown-source.tsx

```diff
@@ -119,7 +65,7 @@ async function showSource(): Promise<void> {
 	sourceButton.classList.add('selected');
 	renderedButton.classList.remove('selected');
 	blurButton(sourceButton);
-	(await source).before(lineActions());
+	(await source).before(await lineActions() as unknown as HTMLDetailsElement);
```

Comment:
The assertions should not be needed, just have the function return the right type